packages/core/src/agent/Agent.ts

```diff
@@ -93,7 +95,7 @@ export class Agent {
     this.messageSender = this.container.resolve(MessageSender)
     this.messageReceiver = this.container.resolve(MessageReceiver)
     this.transportService = this.container.resolve(TransportService)
-    this.wallet = this.container.resolve(InjectionSymbols.Wallet)
+    this.walletService = this.container.resolve(InjectionSymbols.Wallet)
```

Comment:
> It seems a bit weird it's named `walletService` although it's not a service.

Agreed. Do you have any suggestions of the naming?

> What if we expose initPublicDid and publicDid via the wallet "module"?

I would rather not do that at the moment. With the work happening on other did methods and the did module, I'd like to move away from the `initPublicDid` and `publicDid`. With supporting multiple types of dids, it'll be more common to have different kind of public dids, so `publicDid` and `initPublicDid` won't make a lot of sense anymore. So I'd rather not expose that API right now, but rather keep it as is currently (wallet api has only been exposed after 0.1.0 release, so not a breaking change to not make them public).

We can then start looking at how to handle public dids in general through the dids module. The wallet can then focus more on providing the actual crypto instead of dealing with DIDs.

Reply:
I see. Yes, dids module is a better place for this functionality. That can help with `walletService` naming issue. We wouldn't need to have a class member referring to`Wallet` instance.